kylin-spark-project/kylin-spark-engine/src/test/scala/org/apache/kylin/engine/spark/builder/TestCreateFlatTable.scala

```diff
@@ -81,7 +81,8 @@ class TestCreateFlatTable extends SparderBaseFunSuite with SharedSparkSession wi
     //DefaultScheduler.destroyInstance()
   }
 
-  test("Check the flattable spark jobs num correctness") {
+  // todo: [Spark 3.0] BroadcastExchangeExec overwrites job group ID
+  ignore("Check the flattable spark jobs num correctness") {
```

Comment:
modify this test case to :
![image](https://user-images.githubusercontent.com/9430290/120358811-82cda700-c339-11eb-9a78-43efabc299bb.png)

Reply:
test case recovered